src/views/MainPage/MainPage.js

```diff
@@ -27,7 +36,7 @@ function renderMainView() {
       <div class="bubbles">
         <div class="bubble fact">
           <div class="bubble-img">
-            <img src="${FACT_ANIMAL_URL}" alt="the animal the sentence is about">
+            <img src="${FACT_ANIMAL.url}" alt="${FACT_ANIMAL.alt}">
```

Comment:
do not render an image from FACT_ANIMAL constant - take it from Fact object u create

Reply:
I was going to do that after refactoring